.code-samples.meilisearch.yaml

```diff
@@ -150,6 +153,69 @@ getting_started_search_md: |-
   }
   ```
   [About this SDK](https://www.github.com/meilisearch/meilisearch-dotnet)
+getting_started_add_meteorites: |-
+  import json
+
+  json_file = open('meteorites.json')
+  meteorites = json.load(json_file)
+  client.index('meteorites').add_documents(meteorites)
+getting_started_update_rankingRules: |-
+  await client.Index("movies").UpdateRankingRulesAsync(new string[] {
```

Comment:
is see the function have camelCase and snake-case is this normal?

Reply:
Absolutely not! Good catch